scene/gui/option_button.cpp

```diff
@@ -583,9 +600,15 @@ void OptionButton::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("item_focused", PropertyInfo(Variant::INT, "index")));
 }
 
+void OptionButton::set_disable_shortcuts(bool p_disabled) {
+	disable_shortcuts = p_disabled;
+}
+
 OptionButton::OptionButton(const String &p_text) :
 		Button(p_text) {
 	set_toggle_mode(true);
+	set_disable_shortcuts(false);
```

Comment:
This line is optional, `disable_shortcuts` is `false` by default. But I see that you copied this from `MenuButton` which has the same flaw. It's harmless, though.